tools/packaging/static-build/qemu/Dockerfile

```diff
@@ -19,37 +32,40 @@ RUN apt-get update && apt-get upgrade -y && \
 	    bison \
 	    ca-certificates \
 	    cpio \
+	    dpkg-dev \
 	    flex \
 	    gawk \
-	    libaudit-dev \
-	    libblkid-dev \
-	    libcap-dev \
-	    libcap-ng-dev \
-	    libdw-dev \
-	    libelf-dev \
-	    libffi-dev \
-	    libglib2.0-0 \
-	    libglib2.0-dev \
-	    libglib2.0-dev git \
-	    libltdl-dev \
-	    libmount-dev \
-	    libpixman-1-dev \
-	    libselinux1-dev \
-	    libtool \
+	    libaudit-dev${DPKG_ARCH} \
+	    libblkid-dev${DPKG_ARCH} \
+	    libcap-dev${DPKG_ARCH} \
+	    libcap-ng-dev${DPKG_ARCH} \
+	    libdw-dev${DPKG_ARCH} \
+	    libelf-dev${DPKG_ARCH} \
+	    libffi-dev${DPKG_ARCH} \
+	    libglib2.0-0${DPKG_ARCH} \
+	    libglib2.0-dev${DPKG_ARCH} \
+	    libglib2.0-dev${DPKG_ARCH} git \
+	    libltdl-dev${DPKG_ARCH} \
+	    libmount-dev${DPKG_ARCH} \
+	    libpixman-1-dev${DPKG_ARCH} \
+	    libselinux1-dev${DPKG_ARCH} \
+	    libtool${DPKG_ARCH} \
 	    make \
 	    ninja-build \
-	    pkg-config \
-	    libseccomp-dev \
-	    libseccomp2 \
+	    pkg-config${DPKG_ARCH} \
+	    libseccomp-dev${DPKG_ARCH} \
+	    libseccomp2${DPKG_ARCH} \
```

Comment:
Can we install all the toolchains we care about (aarch64, power, and s390x) explicitly?

Reply:
I tried to find all of the cross build lib for aarch64, but I failed. It's hard to find all of them. That's why I use ubuntu MultiArch feature.